styles.css

```diff
@@ -408,3 +413,194 @@ footer textarea {
   padding-left: 10px;
   padding-top: 10px;
 }
+
+@media screen and (min-width: 376px) {
+  header {
+    width: 100%;
+    position: fixed;
+    height: 1.5vh;
+    background-color: #fff;
+  }
+
+  /* navigation bar */
+  nav {
+    width: 100%;
+    justify-content: space-around;
+    background-color: #fff;
+    margin: 0;
+  }
+
+  nav .desk {
+    display: flex;
+    list-style: none;
+  }
+
+  nav li {
+    margin: 0 2em;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  /* introduction part */
+
+  .introduction {
+    background: url(media/intro-background-desk.svg);
+    background-color: #fff;
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 120vh;
+    margin-top: 72px;
+    overflow: visible;
+    border-radius: 0 0 0 150px;
+  }
+
+  .intro {
+    width: 80%;
+    margin: 0 auto 0 18%;
+    display: flex;
+    flex-direction: column;
+    align-self: center;
+  }
+
+  .description {
+    width: 70.5%;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    margin-top: 100px;
+  }
+
+  .solid-welcome {
+    margin-top: 108px;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  /* projetcs */
+
+  .projects {
+    margin-top: 142px;
+  }
+
+  .project {
+    width: 80%;
+    height: auto;
+    flex-direction: row;
+    margin-bottom: 142px;
+  }
+
+  .project div {
+    margin-top: -68px;
+    margin-left: 32px;
+  }
```

Comment:
Please, modify the project cards to match the design as closely as possible. Kindly ensure that the description takes up the correct width as shown on the design. Also the list of technologies are left aligned.

Your design:
![image](https://user-images.githubusercontent.com/20554583/208026184-af412462-6b69-4933-bd8c-6d14b24c1462.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/208026156-2b00471e-d82e-4997-97c1-e85f4e73df48.png)